src/pages/blogger-page/blogger-page.tsx

```diff
@@ -0,0 +1,104 @@
+import { VStack } from '@chakra-ui/react';
+import { useEffect } from 'react';
+import { useLocation, useNavigate, useParams } from 'react-router';
+
+import { PATHS } from '~/app/routes/paths.ts';
+import { BloggerInfo } from '~/components/ui/blogger-info/blogger-info.tsx';
+import { BloggerRecipes } from '~/components/ui/blogger-recipes/blogger-recipes.tsx';
+import { BloggerNotesCollapse } from '~/components/widgets/blogger-notes-collapse/blogger-notes-collapse.tsx';
+import { OtherBlogsSection } from '~/components/widgets/other-blogs-section/other-blogs-section.tsx';
+import { TOAST_MESSAGES } from '~/constants/toast-messages.ts';
+import { useCustomToast } from '~/hooks/use-custom-toast.tsx';
+import {
+    useGetBloggerByIdQuery,
+    useGetBlogsQuery,
+    useGetRecipeByUserIdQuery,
+} from '~/query/services/blogs/blogs-api.ts';
+import { useAppDispatch, useAppSelector } from '~/redux/hooks.ts';
+import { selectIsSubscribedToUser } from '~/redux/selectors.ts';
+import { selectUserId } from '~/redux/slices/auth-slice.ts';
+import { setSubscription, toggleSubscription } from '~/redux/slices/subscriptions-slice.ts';
+
+const { ServerErrorToast } = TOAST_MESSAGES;
+
+export const BloggerPage = () => {
+    const currentUserId = useAppSelector(selectUserId);
+    const { toast } = useCustomToast();
+    const navigate = useNavigate();
+    const { userId } = useParams();
```

Comment:
а может здесь вот так сделаем и не нужно будет постоянные проверки вставлять
```suggestion
    const { userId = '' } = useParams();
```